### PR TITLE
Refactor Part 1

### PR DESCRIPTION
**Pull request by jdisho**
Refactor some code in `Bookmark` group.

### CODE REVIEW

File: Classes/Bookmark/Bookmark.swift [hunk 1]
```diff
@@ -39,12 +39,10 @@ extension Bookmark: Filterable {
...
         let lowerQuery = query.lowercased()
 
-        if type.rawValue.contains(lowerQuery) { return true }
-        if String(number).contains(lowerQuery) { return true }
-        if title.lowercased().contains(lowerQuery) { return true }
-        if owner.lowercased().contains(lowerQuery) { return true }
-        if name.lowercased().contains(lowerQuery) { return true }
-
-        return false
+        return type.rawValue.contains(lowerQuery) ||
```
Comment (BasThomas) [inline Classes/Bookmark/Bookmark.swift]: 🤔, not sure if this is more readable.

File: Classes/Bookmark/BookmarkCell.swift [hunk 2]
```diff
@@ -14,8 +14,8 @@ final class BookmarkCell: SwipeSelectableCell {
 
     static let titleInset = UIEdgeInsets(
         top: Styles.Sizes.rowSpacing,
-        left: Styles.Sizes.icon.width + 2*Styles.Sizes.columnSpacing,
-        bottom: Styles.Text.secondary.preferredFont.lineHeight + 2*Styles.Sizes.rowSpacing,
+        left: Styles.Sizes.icon.width + 2 * Styles.Sizes.columnSpacing,
+        bottom: Styles.Text.secondary.preferredFont.lineHeight + 2 * Styles.Sizes.rowSpacing,
```
Comment (BasThomas) [inline Classes/Bookmark/BookmarkCell.swift]: Neat!

File: Classes/Views/UIView+Extensions.swift [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+//
+//  Created by Joan Disho on 06.06.18.
+//  Copyright © 2018 Ryan Nystrom. All rights reserved.
+//
+
+import Foundation
+
+extension UIView {
+
+    func addSubviews(_ subviews: [UIView]) {
```
Comment (BasThomas) [inline Classes/Views/UIView+Extensions.swift]: I feel like this is overkill.

Comment (BasThomas) [reply Classes/Views/UIView+Extensions.swift]: And if we do want to keep it, some tests / header documentation would be nice.

Comment (jdisho) [reply Classes/Views/UIView+Extensions.swift]: Why overkill? 🤔

I agree for the tests 👍

Comment (SD10) [reply Classes/Views/UIView+Extensions.swift]: Nit: but I don't like how the local variable name `subviews` shadows `self.subviews`

File: Classes/Bookmark/Bookmark.swift [hunk 4]
```diff
@@ -39,12 +39,10 @@ extension Bookmark: Filterable {
...
+            String(number).contains(lowerQuery) ||
+            title.lowercased().contains(lowerQuery) ||
+            owner.lowercased().contains(lowerQuery) ||
+            name.lowercased().contains(lowerQuery)
```
Comment (rnystrom) [inline Classes/Bookmark/Bookmark.swift]: nit: other places in the codebase put the `||` as the first part of the newline

File: Classes/Views/UIView+Extensions.swift (hunk 3 above)
Comment (rnystrom) [reply Classes/Views/UIView+Extensions.swift]: Can we remove this? Unless this is fixing a real problem I don't understand.

**Pull request closed without merging**